### PR TITLE
Speedup drawTile.

### PR DESCRIPTION
**Pull request by jahodfra**
I tried to improve the speed of this function

The most common path has 25% speedup.
Other paths are also bit faster.

I hope the code is not less readable.

### CODE REVIEW

File: src/tic.c [hunk 1]
```diff
@@ -200,62 +200,63 @@ static void drawRectBorder(tic_machine* machine, s32 x, s32 y, s32 width, s32 he
...
 }
 
-static void drawRectRev(tic_machine* machine, s32 y, s32 x, s32 height, s32 width, u8 color)
-{
-	drawRect(machine, x, y, width, height, color);
-}
-
 static void drawTile(tic_machine* machine, const tic_tile* buffer, s32 x, s32 y, u8* colors, s32 count, s32 scale, tic_flip flip, tic_rotate rotate)
 {
+	u8 transparent[16];
```
Comment (nesbox) [inline src/tic.c]: pls use `TIC_PALETTE_SIZE` constant instead `16`

File: src/tic.c [hunk 2]
```diff
@@ -200,62 +200,63 @@ static void drawRectBorder(tic_machine* machine, s32 x, s32 y, s32 width, s32 he
...
 
-	s32 a = flip & tic_horz_flip ? -scale : scale;
-	s32 b = flip & tic_vert_flip ? -scale : scale;
+	if (flip == 0 && rotate == 0 && scale == 1) {
+		// most common path
+		s32 i = 0;
+		for(s32 py=0; py < TIC_SPRITESIZE; py++, y+=1)
+		{
+			s32 xx = x;
+			for(s32 px=0; px < TIC_SPRITESIZE; px++, xx+=1)
```
Comment (nesbox) [inline src/tic.c]: pls change `xx+=1` to `xx++`

File: src/tic.c [hunk 3]
```diff
@@ -200,62 +200,63 @@ static void drawRectBorder(tic_machine* machine, s32 x, s32 y, s32 width, s32 he
...
+
 	flip &= 0b11;
 	rotate &= 0b11;
 
-	s32 a = flip & tic_horz_flip ? -scale : scale;
-	s32 b = flip & tic_vert_flip ? -scale : scale;
+	if (flip == 0 && rotate == 0 && scale == 1) {
+		// most common path
+		s32 i = 0;
+		for(s32 py=0; py < TIC_SPRITESIZE; py++, y+=1)
```
Comment (nesbox) [inline src/tic.c]: pls change `y+=1` to `y++`

**Comment by jahodfra** [comment]
I will do more proper testing and mark the proposed changes. I would like to also see how this improves speed on Raspberry.

**Pull request merged**